src/App.tsx

```diff
@@ -1,61 +1,128 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { Todo, User } from './types/interfaces';
+import { TodoList } from './components/TodoList';
+
+export function addUserToTodo(todos: Todo[], users: User[]): Todo[] {
+  return todos.map(todo => {
+    return {
+      ...todo,
+      user: users.find(user => user.id === todo.userId) || undefined,
+    };
+  });
+}
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(
+    addUserToTodo(todosFromServer, usersFromServer),
+  );
+
+  const [userValue, setUserValue] = useState<string>('0');
+
+  const [title, setTitle] = useState<string>('');
+
+  const [errors, setErrors] = useState<{ title: boolean; user: boolean }>({
+    title: false,
+    user: false,
+  });
+
+  const handleTitle = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    setTitle(event.target.value);
+    if (errors.title) {
+      setErrors(prevErrors => ({ ...prevErrors, title: false }));
+    }
+  };
+
+  const handleUserValue = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ): void => {
+    setUserValue(event.target.value);
+
+    if (errors.user) {
+      setErrors(prevErrors => ({ ...prevErrors, user: false }));
+    }
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>): void => {
+    event.preventDefault();
+
+    if (!title || userValue === '0') {
```

Comment:
<img width="564" alt="image" src="https://github.com/user-attachments/assets/cb8f32e9-4204-45ed-b73a-df2ed5172f9d">
ментор написав, що потрібно пофіксити проблему зі створенням туду в якому тайтл є лише пробіли.




для цього, як написав ментор треба викликати метод `.trim()` який видалить усі зайві пробіли
```suggestion
    if (!title.trim() || userValue === '0') {
```